spec/jobs/duplicate_jobs/session_spec.rb

```diff
@@ -99,4 +99,82 @@
       expect(cloned_questions).to eq(['Question::Single'])
     end
   end
+
+  context 'when duplicating session with reflection' do
```

Comment:
What do you think about adding tests to detect if the source is the same in session_clone speck?